shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/Favorite.kt

```diff
@@ -2,7 +2,11 @@ package com.mbta.tid.mbta_app.model
 
 import kotlinx.serialization.Serializable
 
-@Serializable public data class Favorites(val routeStopDirection: Set<RouteStopDirection>? = null)
+@Serializable
+public data class Favorites(val routeStopDirection: Set<RouteStopDirection>? = null) {
+    public fun isFavorite(rsd: RouteStopDirection): Boolean =
```

Comment:
```suggestion
    public fun isFavorite(@ObjCName(swiftName = "_") rsd: RouteStopDirection): Boolean =
```

This will remove the need for the FavoritesExtension.

Reply:
Ahh, I was trying to do that with an empty string and it was failing, so I assumed it wasn't possible, nice!